### ISSUE TITLE
Computed Watchers Not Firing

### ISSUE THREAD

**Issue by grayfella** [vuejs/vue#8409]
### Version
2.5.16

### Reproduction link
[https://vuejs.org/v2/guide/computed.html#Watchers](https://vuejs.org/v2/guide/computed.html#Watchers)

### Steps to reproduce
Use the demo for entering a question on the Docs page using a Amazon Kindle 7'' or Android Tablet (I used a Lenovo Tab 3 7'' version)
https://vuejs.org/v2/guide/computed.html#Watchers

### What is expected?
When typing in the field the value is evaluated after each letter is tapped when using a browser on a tablet, as per the demo instructions. 

### What is actually happening?
Nothing is evaluated until the space bar is tapped or a question mark / exclamation mark is selected from the keyboard. 

---
I use this to create a drop down list of options when a user starts typing into a text field using Android tablet specifically Amazon Fire.
The user starts typing and after letters are entered the list is shown..
After updating then the list is only shown if the user taps the space bar or the enter key on the on screen keyboard.
This works as expected when using a browser on a computer.
I have tested my code on Mac using Safari, Chrome and Firefox and no errors are in the console. 
On Amazon Fire and it works as expected when on version 2.5.13
Updating my VueJS to version 2.5.16 and this stops working on the tablet and requires the space key, exclamation mark, question mark or enter key to be tapped for the evaluation to fire/run when using the tablet device.

**Comment by zaggy** [comment]
hi, may I try to fix this as my first bug?

**Comment by sodatea** [comment]
@zaggy Sure! 😀

**Comment by grayfella** [comment]
Just a guess but would it be this issue that was fixed that causes it? https://github.com/vuejs/vue/pull/7121 
I'm not sure how I would exclude that from a version for testing.

**Comment by THEozmic** [comment]
What's the status of this?

**Comment by karanisverma** [comment]
@zaggy are you working on it? if not I would like to pick this up! 🙂

**Comment by chriswburke** [comment]
is this still an issue?

**Comment by grayfella** [comment]
Yes it is. I'm using the standard help page / demo page on a Kindle Fire and you have to tap the enter key when typing letters before the check is made. A question mark or an exclaimation mark after a word does fire the check the same as tapping the enter key.

**Comment by yyx990803** [comment]
Just a note this may have to do with `v-model` preventing temporary updates during a composition (which is necessary for some input methods e.g. Chinese) - and some devices may be firing composition events even for normal English keyboards. That said I don't have a Kindle Fire so it's hard for me to reproduce.

**Comment by grayfella** [comment]
I could replicate also on a Lenovo Tab 3. On both I have spell check / suggestions disabled in off chance that would help but it doesn't.

**Comment by Stian-Scholtz** [comment]
@zaggy Did you fix you first bug? Been a while and I am still experiencing this issue.

**Comment by mouradsme** [comment]
Does this problem exist for all Android devices? (I'm using an online emulator running on CentOS, and it's kinda working fine, although I wouldn't really count on results given by emulators)

**Comment by grayfella** [comment]
It okay on an old Android 5 device I tested with but the Kindle Fires have the issue. Even with all the spelling options disabled there is an underline on the text you type in the field. Once you press space bar or enter that line goes and then the event is fired and in my case the dropdown shown.

**Comment by NazimAli2017** [comment]
Is this reproducible with Android emulation? If so, would be a fun bug to work on. Had a similar issue with an application for work.

**Comment by Obinna-Nwankwo** [comment]
Hope you've fix the bug
